.buildkite/scripts/run-tests.sh

```diff
@@ -8,11 +8,14 @@ with_mage
 with_go_junit_report
 
 set +e
-mage -debug test | tee tests-report-linux.txt
+mage -debug test > tests-report-linux.txt
 exit_code=$?
-
 set -e
 
+# Buildkite collapse logs under --- symbols
+# need to change --- to anything else or switch off collapsing (note: not available at the moment of this commit)
+awk '{gsub("---", "----"); print }' tests-report-linux.txt
```

Comment:
Could you try to make it fail some test ? 
As this file is used by `go-junit-report` to be sure that it is everything running as expected @leo-ri

Reply:
Thanks for checking this out @leo-ri 
https://github.com/elastic/package-registry/pull/1057/commits/8a63e8c60a3eb5f720cb26d13f26d0828ca52689
https://buildkite.com/elastic/package-registry/builds/284#0189643b-cc85-44f0-bb89-d2f29fae56d8